config/fieldparams/minimal.go

```diff
@@ -28,6 +28,6 @@ const (
 	MaxWithdrawalsPerPayload              = 4             // MaxWithdrawalsPerPayloadLength defines the maximum number of withdrawals that can be included in a payload.
 	MaxBlobsPerBlock                      = 6             // MaxBlobsPerBlock defines the maximum number of blobs with respect to consensus rule can be included in a block.
 	MaxBlobCommitmentsPerBlock            = 16            // MaxBlobCommitmentsPerBlock defines the theoretical limit of blobs can be included in a block.
-	BlobLength                            = 4             // BlobLength defines the byte length of a blob.
-	BlobSize                              = 128           // defined to match blob.size in bazel ssz codegen
+	BlobLength                            = 131072        // BlobLength defines the byte length of a blob.
```

Comment:
both of blob length and blobsize are the same number?

Reply:
Yeap, they mean the same thing afaik